apps/ledger-live-mobile/src/logic/getWindowDimensions.android.ts

```diff
@@ -1,7 +1,9 @@
-import ExtraDimensions from "react-native-extra-dimensions-android";
+import { Dimensions } from "react-native";
 
-export default () => ({
-  width: ExtraDimensions.get("REAL_WINDOW_WIDTH"),
-  height: ExtraDimensions.get("REAL_WINDOW_HEIGHT"),
-});
-export const softMenuBarHeight = () => ExtraDimensions.get("SOFT_MENU_BAR_HEIGHT");
+export default () => {
+  const screen = Dimensions.get("screen");
+  return {
+    width: screen.width,
+    height: screen.height,
+  };
```

Comment:
We could directly return screen here to avoid new allocations
```suggestion
  return screen;
```

We could also rename this as getScreenDimensions to not confuse with `Dimensions.get("window")` but it means changing a lot of imports and I understand if we want to keep the PR smaller

Reply:
Indeed but for IOS it uses window because it's not the same behaviour on both platforms
So it will be confusing also with IOS 

It's explained here https://stackoverflow.com/a/75234700